### ISSUE TITLE
Add preconditions to invoke WFS 2.0 from DGIWG WFS profile

### ISSUE THREAD

**Issue by rjmartell** [opengeospatial/ets-dgiwg#4]
The latest revision of the DGIWG WFS 2.0 profile ([DGIWG 122](https://www.dgiwg.org/dgiwg/htm/documents/standards_implementation_profiles.htm), 16 Nov 2015) is based on the WFS 2.0.2 specification ([OGC 09-025r2](http://docs.opengeospatial.org/is/09-025r2/09-025r2.html)).

There should be a precondition to check that the IUT implements WFS 2.0.2. If not, all remaining tests are skipped.

**Comment by rjmartell** [comment]
Defining preconditions as actual test methods in a separate conformance class is preferred, since this makes them much more visible in the report so it's obvious if one fails.

For example, see the `org.opengis.cite.iso19142.SuitePreconditions` class in the WFS test suite.This class is referenced in the TestNG configuration file as the only member of the "Preconditions" conformance class: 

```
<test name="Preconditions">
    <classes>
      <class name="org.opengis.cite.iso19142.SuitePreconditions"/>
    </classes>
  </test>
```

**Comment by dstenger** [comment]
We are currently fixing this issue.

**Comment by dstenger** [comment]
In the last meeting we decided to implement this check in a single test to prevent failure of the whole test suite when WFS 2.0.0 is used.

**Comment by dstenger** [comment]
A test for WFS 2.0.2 was implemented in pull request #6 (more information in the pr).

The pull request has already been merged to the master branch.
